pkg/ddevapp/list.go

```diff
@@ -14,29 +14,53 @@ import (
 	"time"
 )
 
+// ListCommandSettings conains all filters and settings of the `ddev list` command
+type ListCommandSettings struct {
```

Comment:
This is a significant code improvement, thanks!

Reply:
I thought it's a way to get rid of all these variables for options.

Reply:
If that works then we can improve also the other commands.

Reply:
I don't like to use those statics/globals at all. Mostly newer commands use the if changed stuff.